frontend/test/metabase/scenarios/dashboard-filters/reproductions/24500-gracefully-deal-with-corrupted-filter.cy.spec.js

```diff
@@ -60,14 +67,14 @@ describe("issue 15279", () => {
               visualization_settings: {},
               parameter_mappings: [
                 {
-                  parameter_id: firstFilter.id,
+                  parameter_id: listFilter.id,
                   card_id,
-                  target: ["dimension", ["field-id", PEOPLE.SOURCE]],
+                  target: ["dimension", ["field", PEOPLE.SOURCE, null]],
                 },
                 {
-                  parameter_id: secondFilter.id,
+                  parameter_id: searchFilter.id,
                   card_id,
-                  target: ["dimension", ["field-id", PEOPLE.NAME]],
+                  target: ["dimension", ["field-id", PEOPLE.NAME, null]],
```

Comment:
Syntax for this line should be `target: ["dimension", ["field", PEOPLE.NAME, null]],`

Reply:
Great catch, @daltojohnso.
That's a leftover which I overlooked. Thanks